doc/src/manual/control-flow.md

```diff
@@ -567,17 +574,16 @@ below all interrupt the normal flow of control.
 For example, the [`sqrt()`](@ref) function throws a [`DomainError`](@ref) if applied to a negative
 real value:
 
-```julia
+```jldoctest
 julia> sqrt(-1)
 ERROR: DomainError:
 sqrt will only return a complex result if called with a complex argument. Try sqrt(complex(x)).
- in sqrt(::Int64) at ./math.jl:278
```

Comment:
> What do other people have to say?

I've got no particular preference for either, largely depends on the purpose of each doctest I think.